index.html

```diff
@@ -0,0 +1,330 @@
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <section class="contact flex" id="contact">
+        <p class="parag text-dark">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          class="flex"
+          action="https://formspree.io/f/xdobralq"
+          method="post"
+          id="mail"
+        >
+          <input
+            name="fname"
+            id="name"
+          />
+          <input
+            name="email"
+            type="text"
+            required
+            placeholder="Email address"
+            id="email"
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+          ></textarea>
+          <button class="special" type="submit">Get in touch</button>
+        </form>
+      </section>
+    </main>
+
+    <script
+      src="https://kit.fontawesome.com/e966a3bd22.js"
+      crossorigin="anonymous"
+    ></script>
+  </body>
+</html>
```

Comment:
**Project Requirements**
> The section you need to build for this milestone is:
>  - the contact form:
> - and the footer (if present in the design)

- [ ] I suggest putting a footer in your project as the template of your choosing also has a footer in its design

<img width="199" alt="Screenshot 2023-02-11 at 12 48 25 AM" src="https://user-images.githubusercontent.com/44798044/218187141-a7f6d12e-8cc7-4ddc-8a74-712fc67043b1.png">


![require change](https://user-images.githubusercontent.com/44798044/218188459-82eb964f-f886-4715-896b-f8df7d7f9e6b.png)